src/base-client.ts

```diff
@@ -14,6 +14,12 @@ export class BaseSlackAPIClient implements BaseSlackClient {
     this.#baseURL = options.slackApiUrl || "https://slack.com/api/";
   }
 
+  setSlackApiUrl(apiURL: string) {
```

Comment:
My only suggestion is to add a JSDoc description here. It would also be good to explain what form the URL should be. Based on the default fallback on line 14, we want it to:
- include the protocol
- include the path
- have a trailing slash

All these things are little pitfalls that might trip users up.. perhaps may want to even check for these things inside this method? Not sure. Might be overkill. But at a minimum, a JSdoc that explains the expectations around the parameter would be great, helpful and probably enough guidance for most, I think.

Reply:
I added some docs and decided to automatically set that trailing slash when we make api calls. Added some tests for that too.